dockerclient.go

```diff
@@ -258,6 +258,37 @@ func (client *DockerClient) ContainerChanges(id string) ([]*ContainerChanges, er
 	return changes, nil
 }
 
+func (client *DockerClient) ContainerStats(id string, stopChan <-chan struct{}) (<-chan StatsOrError, error) {
+	uri := fmt.Sprintf("/%s/containers/%s/stats", APIVersion, id)
+	req, err := http.NewRequest("GET", client.URL.String()+uri, nil)
```

Comment:
tiny nit:  since you're doing a simple GET, client.HTTPClient.Get(uri) might be marginally cleaner.